consensus/replay.go

```diff
@@ -246,15 +246,17 @@ func (cs *ConsensusState) startForReplay() {
 	cs.BaseService.OnStart()
 
 	// since we replay tocks we just ignore ticks
-	go func() {
-		for {
-			select {
-			case <-cs.tickChan:
-			case <-cs.Quit:
-				return
+	// TODO:!
```

Comment:
mostly from laziness. I don't think anyones used the replay console since I wrote it, and it has no tests or documentation. Might as well remove it from the cli for this release and bring it back when we have time to write tests for it (it would make for a fantastic blog post about how tendermint works!)